cmd/chaininfo/arbitrum_chain_info.json

```diff
@@ -44,6 +44,7 @@
       "sequencer-inbox": "0x1c479675ad559dc151f6ec7ed3fbf8cee79582b6",
       "validator-utils": "0x9e40625f52829cf04bc4839f186d621ee33b0e67",
       "validator-wallet-creator": "0x960953f7c69cd2bc2322db9223a815c680ccc7ea",
+      "stake-token": "0x0000000000000000000000000000000000000000",
```

Comment:
This (and nova's) will be WETH right? Could we fill in these addresses now so new nitro releases will theoretically support bold on arb1/nova?